contributions/contribution.yaml

```
@@ -299,6 +299,86 @@ paths:
           description: Invalid input
         500:
           description: Internal error
+  /contributions/{id}/talents/{talentid}:
```

```suggestion
  /contributions/{id}/talents/{talent_id}:
```
Could we use this approach for better readability? Similarly, for the other endpoint. Thanks.